components/Grid/grid.css

```diff
@@ -0,0 +1,47 @@
+.grid {
+  background: var(--light-gray);
+  border-radius: 0.2rem;
+  padding: 1rem;
+}
```

Comment:
don't mix documentation css with component's css

Reply:
that component css only

Reply:
Isn't this the css for the container for your component?.... `background: var(--light-gray);
  border-radius: 0.2rem;
  padding: 1rem;`
  
  How are these properties affecting the grid?